index.html

```diff
@@ -11,7 +11,7 @@
             window.onload = function () {
                 // Begin Swagger UI call region
                 const ui = SwaggerUIBundle({
-                    url: "./builds/OpenSearch.latest.yaml", //Location of Open API spec in the repo
+                    url: "https://github.com/opensearch-project/opensearch-api-specification/releases/download/latest-main-release/OpenSearch.latest.yaml",
```

Comment:
As the build workflow uses `automatic_release_tag: latest-${{ steps.branch.outputs.name }}` this should be `latest-main` rather than `latest-main-release`
```suggestion
                    url: "https://github.com/opensearch-project/opensearch-api-specification/releases/download/latest-main/OpenSearch.latest.yaml",
```

Reply:
Fixed.